### DISCUSSION TITLE
How can I reset the admin password?

### DISCUSSION THREAD

**Discussion by jobayer977** [logto-io/logto#5895] [General]
Somehow, I forgot my admin password. How can I reset it? Is there any way?

**Comment by xiaoyijun** [comment]
Hi @jobayer977 , are you using the Logto cloud? If yes, you can reset your password from the sign-in page if you have bind your phone number or email address. If you're using the oss version, currently we don't support to reset the admin password, but that's a good point, maybe we need to support it for OSS version. @fleuraly

**Reply by jobayer977** [reply]
No, I'm using the OSS version. It's good to know that the cloud version supports password resets. Adding this feature to the OSS version would definitely be beneficial.

**Reply by AlliotTech** [reply]
Hi guys, is there any updates about this topic?

**Comment by charIeszhao** [comment]
If it's OSS, then one easier option could be directly modifying the DB value. Just simply copy and paste an existing password hash string from another user account to the admin user.

**Reply by monolithed** [reply]
That’s why I stopped using Sonatype Nexus. Directly modifying the DB with an unknown structure is always risky — you can break everything with no guarantee that the table structure won’t change in the future. On top of that, it requires instructions, training, and every time it's a half-day ordeal.
